packages/ai-jsx/src/core/errors.ts

```diff
@@ -20,6 +20,10 @@ export enum ErrorCode {
   NestedAIUIStreamsAreNotSupported = 1015,
   UnknownUIComponentId = 1016,
   UnknownSerializedComponentType = 1017,
+  AnthropicDoesNotSupportCompletionModels = 1018,
```

Comment:
This seems likely to get out of hand. Maybe just `CompletionModelNotSupported` that can be used for any provider?

Reply:
Good question. With the error codes already here, I had to choose between:

1. Error codes reflect semantic situations and may be emitted from multiple callsites.
2. Error codes reflect specific callsites.

Ultimately, I went with (2), because I thought that would make it easier to track down what was happening with bug reports. But it does have the downside you're identifying here. We very well may feel that this gets out of hand and reshuffle down the line. Or maybe we should do it now.